src/data/faq.json

```
@@ -116,6 +116,62 @@
                     }
                 ]
             },
+            {
+                "title": "Testzertifikat",
+                "id": "test_cert",
+                "accordion": [
+                    {
+                        "title": "What is the purpose of the digital COVID test certificate?",
+                        "anchor": "test_cert_why",
+                        "active": true,
+                        "textblock": [
+                            "Starting with version 2.4 of the Corona-Warn-App (CWA), you can request a digital test certificate via the app.  The new feature allows you to prove your negative test result through an official, digital COVID test certificate in the form of a QR code in the CWA.  Within the countries of the European Union, it can serve as officially valid proof of a negative test result, which you can use, for example, for travel or in other legally prescribed cases, such as a visit to a restaurant. You can then show the QR code in the CWA."

```

Some double spaces here.

```suggestion
                            "Starting with version 2.4 of the Corona-Warn-App (CWA), you can request a digital test certificate via the app. The new feature allows you to prove your negative test result through an official, digital COVID test certificate in the form of a QR code in the CWA. Within the countries of the European Union, it can serve as officially valid proof of a negative test result, which you can use, for example, for travel or in other legally prescribed cases, such as a visit to a restaurant. You can then show the QR code in the CWA."
```